workflow-templates/dreact-ci.yml

```diff
@@ -0,0 +1,62 @@
+name: DReact CI
+on:
+  pull-request:
+    types: [closed]
```

Comment:
Should we add `branches: [ $default-branch ]` as a sensible default or do you think it's better to leave this open-ended?

Reply:
I think setting a default makes sense!